packages/zowe-explorer/src/job/actions.ts

```diff
@@ -73,10 +73,16 @@ export async function getSpoolContent(session: string, spool: zowe.IJobFile, ref
     }
     await profiles.checkCurrentProfile(zosmfProfile);
     if (profiles.validProfile === ValidProfileEnum.VALID || profiles.validProfile === ValidProfileEnum.UNVERIFIED) {
-        const uri = toUniqueJobFileUri(session, spool)(refreshTimestamp.toString());
+        const uri: vscode.Uri = toUniqueJobFileUri(session, spool)(refreshTimestamp.toString());
         try {
             await vscode.window.showTextDocument(uri);
         } catch (error) {
+            const isTextDocActive =
+                vscode.window.activeTextEditor.document.uri?.path === `${spool.jobname}.${spool.jobid}.${spool.ddname}`;
+
+            if (String(error.message).includes("Failed to show text document") && isTextDocActive) {
```

Comment:
Will this work if the user works in a different language (as Theia is now localized)?

Reply:
You have a good point here, might have to look into how I could fetch that error localized and check for it...

Reply:
Check if vscode api defines that error as an error code and then check against that code. The message string will likely change in the future.